wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -32,6 +32,7 @@ class CreateSecureGuestLinkViewController: UIViewController, CreatePasswordSecur
     weak var delegate: ValidatedTextFieldDelegate?
 
     private lazy var viewModel: CreateSecureGuestLinkViewModel = {
+        // Inject the useCase here, have the userSession, get the factory and inject it
```

Comment:
Is this a todo?

Reply:
Nope, it's already done, so I'm planning to remove the comment.

Reply:
Done!